gradle/libs.versions.toml

```diff
@@ -19,6 +20,7 @@ buildconfig = "5.5.1"
 [libraries]
 junit-jupiter = { module = "org.junit.jupiter:junit-jupiter", version.ref = "junit-jupiter" }
 junit-platform-launcher = { module = "org.junit.platform:junit-platform-launcher" }
+assertj = { module = "org.assertj:assertj-core", version.ref = "assertj" }
```

Comment:
thanks for introducing assertJ dependency. Sensible usage of it will enhance testing code readability dramatically!